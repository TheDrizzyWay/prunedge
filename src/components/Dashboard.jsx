import React from 'react';

const Dashboard = () => {
    return (
        <div className="dashboard">
            <p>Dashboard</p>
            <div className="filler-1"></div>
            <div className="dashboard__info">
                <div className="dashboard__info--statistics">
                    <div className="dashboard__info--stat">
                        <div className="stat-icon stat-icon-purple">
                            <i className="fas fa-sitemap"></i>
                        </div>
                        <div className="stat-text">
                            <p>Departments</p>
                            <h4>5 members</h4>
                        </div>
                    </div>

                    <div className="dashboard__info--stat">
                        <div className="stat-icon stat-icon-blue">
                            <i className="fas fa-user"></i>
                        </div>
                        <div className="stat-text">
                            <p>Members</p>
                            <h4>24 members</h4>
                        </div>
                    </div>

                    <div className="dashboard__info--stat">
                        <div className="stat-icon stat-icon-turquoise">
                            <i className="fas fa-users"></i>
                        </div>
                        <div className="stat-text">
                            <p>Units</p>
                            <h4>3 Units</h4>
                        </div>
                    </div>

                    <div className="dashboard__info--stat">
                        <div className="stat-icon stat-icon-green">
                            <i className="fas fa-archive"></i>
                        </div>
                        <div className="stat-text">
                            <p>Documents</p>
                            <h4>12,097 files</h4>
                        </div>
                    </div>
                </div>
                <button className="new-document-btn">
                    <i className="fas fa-plus"></i> <p>New Document</p>
                </button>
            </div>
        </div>
    );
}

export default Dashboard;
