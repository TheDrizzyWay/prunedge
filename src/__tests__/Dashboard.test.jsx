import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Dashboard from '../components/Dashboard';

configure({ adapter: new Adapter() });

describe('Dashboard component', () => {
  it('should render dashboard', () => {
    const wrapper = shallow(
      <Dashboard />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
