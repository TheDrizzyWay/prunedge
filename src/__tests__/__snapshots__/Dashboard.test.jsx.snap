// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard component should render dashboard 1`] = `ShallowWrapper {}`;
